assets/js/wcmp-frontend.js

```diff
@@ -368,16 +397,21 @@ jQuery(function($) {
     },
 
     /**
-     * Check if the currently selected shipping method is allowed to have delivery options.
+     * Check if the currently selected shipping method is allowed to have delivery options by checking if the name
+     *  starts with any value in a list of shipping methods.
+     *
+     * Most of the values in this list will be full shipping method names, with an instance id, but some can't have one.
+     *  That's the reason we're checking if it starts with this value instead of whether it's equal.
      *
```

Comment:
Uitlijning van de command (`starts with` en `That's the` )

Reply:
Zelfde verhaal als bovenstaand